rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,102 @@
+  * technical side - All php-src changes excluding user facing changes. It covers alse the implementation of user facing changes.
+  * TC - PHP Technical Committee.
+  * user facing changes - Changes in PHP languege, user visible funtions, classes, interfaces, constants and other user visible constructs.
+
+==== Work flow ====
+
+The work flow for TC is following:
+
+  - Any core developer can add label tc-hold on issue or pull request. 
+  - The core developer should mention the TC GitHub team to notify members of the TC.
+  - The TC discusses the issue or pull request and vote on it (this can be done privately)
+    * The TC should consult with maintainers if the issue / PR changes or impacts code maintained by the maintainer
+    * The TC should consult with RM if the issue / PR is about classification of the self contained feature or a bug
+  - The TC announce its decision and reasoning by commenting on the issue or pull request
+  - The core developer will follow the decision which can mean (not limited to):
+    * merging the issue / PR
+    * closing / rejecting the issue / PR  
+    * making requested change to the PR
+
+It should be noted that if the problem is with a specific commit, the core developer can create an issue specifying the
+problem and put label on it.
+
+==== Decision process ====
+
+=== RFC ===
+
+The RFC process will continue to be used for user facing changes. It will also continue to be used for all processing
+changes. However, it will no longer be used for any technical decisions.
+
+=== TC ===
+
+The TC members will only consider technical aspects. Specifically the decision is not made for the user facing changes
+which continue to be decided by the RFC process. TC can however decide on the implementation of those features.
+
+The TC does not need to follow technical decisions made previously by the RFC process.
+
+The TC can decide whether the issue / PR is a feature or a bug if there is a disagreement between RM and a core
+developer. If it is a feature, the TC can also decide whether it is a self contained feature by assessing its technical
+impact on php-src. This happens only on request like described in the work flow section.
+
+The way how to discuss and vote on the issue is up to the TC members. The only requirement is that the decision needs
+to be communicated publicly by commenting on GitHub issue or pull request.
+
+==== TC membership ====
+
+The TC will be composed of 5 member who will be voted once a year together with release managers by the RFC process.
+It will be done using a single transferable vote if more than 5 candiates volunteer. The candidate must be a core
+developer and should have a general knowledge about PHP internals.
+
+If one of the members wants or needs to drop out of being part of the TC, the new volunteer will be selected using
+the RFC process. The vote happens only if there are more than one volunteer. The selection is automatic if there is
```

Comment:
Don't you mean STV process?  An RFC to replace someone seems... odd.

Reply:
I think I see the intent here, but it could use rewording.  There's also a lot of squish here; how long should the "call for candidates" be for a vacancy RFC?  Is it different than for the annual election?  The process could use a lot more precision.

Reply:
Yeah it's kind of re-election if someone decides not to carry on in TC. I'm wondering if it's better to just do nothing and leave TC with less members untill the next election. If you have any idea what would be best here and wording for that, it would be apprecieated.

Reply:
I updated this step to allow TC working in reduced number unless it's below 3 till the next election. Think that makes most sense.